### ISSUE TITLE
Argos Comics (PT-Br Source) Error

### ISSUE THREAD

**Issue by EvaldoCeBr** [keiyoushi/tachiyomi-extensions-source#4815]
### Source information

Argos Comics 1.4.40

### Source language

Portuguese Brazilian 

### Steps to reproduce

When I open the source, it always gives an error. Only after entering WebView does it return to normal.

![Screenshot_2024-08-28-17-36-33-952_app mihon](https://github.com/user-attachments/assets/b2fa854c-ecc3-4c40-a6db-53d9d3ef74aa)

### Expected behavior

It doesn't give an error.

### Actual behavior

It's giving an error.

### Mihon/Tachiyomi version

0.16.5

### Android version

Android 12

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by Smol-Ame** [comment]
Duplicate of #4812
